skorch/callbacks/training.py

```diff
@@ -273,3 +278,145 @@ def _sink(self, text, verbose):
         #  We do not want to be affected by verbosity if sink is not print
         if (self.sink is not print) or verbose:
             self.sink(text)
+
+
+
+class ParamMapper(Callback):
+    """Map arbitrary functions over module parameters filtered by pattern
+    matching. In the simplest case the function is only applied once at
+    the beginning of a given epoch (at ``on_epoch_begin``) but more complex
+    execution schemes (e.g. periodic application) are possible using
+    ``at`` and ``scheduler``.
+
+    Examples
+    --------
+    Initialize a layer on first epoch before the first training step:
+
+    >>> init = partial(torch.nn.init.uniform_, a=0, b=1)
+    >>> cb = ParamMapper('linear*.weight', at=1, fn=init)
+    >>> net = Net(myModule, callbacks=[cb])
+
+    Reset layer initialization if train loss reaches a certain value:
+
+    >>> at = lambda net: net.history[-1, 'train_loss'] > 1
```

Comment:
Does this fit the description that the function is triggered `if train loss reaches a certain value`?

Reply:
Yes. The `at` function returns `True` as soon as `train_loss` reaches a certain value (i.e. `1`).

Reply:
I would assume that `train_loss` decreases over time, so at the start, the condition is always true and later false. The description makes it sound like the condition is false at the start and becomes true over time.